src/tests/ftest/launch.py

```diff
@@ -564,7 +564,8 @@ def main():
     test_files = get_test_files(test_list)
 
     # Run all the tests
-    exit(run_tests(test_files, tag_filter, args))
+    run_tests(test_files, tag_filter, args)
+    exit(0)
```

Comment:
As discussed on IM, we just want to mask (i.e. bitwise-or) out a return code of 1 and 8 from the `avocado` exit status.

Reply:
This has been implemented.